modules/ROOT/nav.adoc

```diff
@@ -20,6 +20,7 @@ include::{partialsdir}/attributes.adoc[]
 
 // SECURITY
 * xref:security/index.adoc[{device-security-service}]
+** xref:security/coding.adoc[Coding]
```

Comment:
There should not be a coding section

Reply:
After speaking with @finp we've decided to add it.